### DISCUSSION TITLE
Chat Channels

### DISCUSSION THREAD

**Discussion by ZEROOblivious** [Zrips/CMI#3054] [Feature Requests]
Global Chat / Other Chat 

Separation of two chat channels, with functionality of DISCORDSRV detecting the chat channel you're in with a different Emoji or something. 

---
### Detailed explanation of what it should do, when and in what situation: ###
/global  <-- Toggles normal Chat 
/chat channel (channel name)

cmi.global.chat
cmi.global.bypass
cmi.channel.bypass
cmi.channel.X
### Basic explanation as to why this feature is needed and/or useful to have: ###

Separation of two chat Rooms, to where you can have two Categories talking at the same time.
People would be talking in Global while others are talking in a different format. 

We have a roleplay server so in my case

/global would be the default normal chat that everyone can talk in
/channel rp would be a Roleplay chat with a prefix but DiscordSRV still picks it up/sees it.

**Comment by ZEROOblivious** [comment]
Bump

**Comment by beamer91** [comment]
We need this
